packages/routing/cardstack/static-model.js

```diff
@@ -1,99 +1,5 @@
 module.exports = function() {
   return [{
-    type: 'content-types',
-    id: 'spaces',
```

Comment:
Why did this stuff need to move into bootstrap-schema?

Reply:
sorry, that was a change that I made when I was working thru throwing errors when schema stuff appeared in the static-model feature, but then realized that its too soon to start enforcing that. I'll move these things back.